code/modules/mob/living/carbon/human/human.dm

```diff
@@ -922,10 +923,13 @@
 		remoteview_target = null
 		reset_view(0)
 
-/mob/living/carbon/human/proc/get_visible_gender()
+/mob/living/carbon/human/get_visible_gender()
 	if(wear_suit && wear_suit.flags_inv & HIDEJUMPSUIT && ((head && head.flags_inv & HIDEMASK) || wear_mask))
-		return NEUTER
-	return gender
+		return PLURAL //plural is the gender-neutral default
+	if(species)
+		if(species.ambiguous_genders)
+			return PLURAL // regardless of what you're wearing, your gender can't be figured out
+	return get_gender()

```

Comment:
Sneaking in a comment from another code base, but this could probably be `..()` instead, to rely on however the base `/get_visible_gender()` acquires the appropriate gender.

Reply:
That would change one proc call to two, unfortunately. Making the base `get_visible_gender()` use the ternary `get_gender()` uses would reduce it to just one proc, but also open the risk of breaking compatibility if get_gender ever uses different logic.